Resources/Prototypes/_NF/Catalog/Fills/Paper/misc.yml

```diff
@@ -1,7 +1,40 @@
 - type: entity
   id: PaperWrittenPirateScribbles
-  suffix: pirate scribbles
+  name: missing soda
+  suffix: MappedPaper
   parent: Paper
   components:
   - type: Paper
     content: book-text-pirate-scribbles
+
+- type: entity
+  id: PaperWrittenFrontierFridge
+  name: fridge policy
+  suffix: MappedPaper
+  parent: Paper
+  components:
+  - type: Paper
+    stampState: paper_stamp-centcom
+    stampedBy:
+    - stampedColor: '#004200FF'
+      stampedName: Sergeant Stamps-the-Papers
+      stampType: RubberStamp
```

Comment:
If Centcomm gives out personalized stamps, fine by me, it's their business, but otherwise, how about this?

```suggestion
    - stampedColor: '#004200FF'
      stampedName: Central Admiralty of Frontier
      stampType: RubberStamp
    - stampedColor: '#333333FF'
      stampedName: Stamps-the-Papers
      stampType: Signature
```

In-game (signature placement is random, can look worse)
![image](https://github.com/new-frontiers-14/frontier-station-14/assets/166147148/4394ebc5-6cb0-48d7-aeb3-45933944fc20)